spec/lib/paths_for_educator_spec.rb

```diff
@@ -20,28 +20,7 @@ def navbar_links(educator)
       end
     end
 
-    context 'when HOUSEMASTERS_AUTHORIZED_FOR_GRADE_8 disabled' do
-      before { @housemasters_authorized_for_grade_eight = ENV['HOUSEMASTERS_AUTHORIZED_FOR_GRADE_8'] }
-      before { ENV['HOUSEMASTERS_AUTHORIZED_FOR_GRADE_8'] = nil }
-      after { ENV['HOUSEMASTERS_AUTHORIZED_FOR_GRADE_8'] = @housemasters_authorized_for_grade_eight }
-
-      it 'respects PerDistrict for /educators/my_students' do
-        expect(navbar_links(pals.shs_harry_housemaster)).to eq({
-          school: '/schools/shs',
-          absences: '/schools/shs/absences',
-          tardies: '/schools/shs/tardies'
-        })
-      end
-    end
-
```

Comment:
This isn't needed anymore, these features have been turned on and are set in `development.rb` and `test.rb`.